Lib/fontTools/feaLib/ast.py

```diff
@@ -319,21 +319,20 @@ def addDefinition(self, definition):
         assert isinstance(definition, MarkClassDefinition)
         self.definitions.append(definition)
         for glyph in definition.glyphSet():
-            if glyph in self.definitions:
-                otherLoc = self.definitions[glyph].location
-                raise FeatureLibError(
-                    "Glyph %s already defined at %s:%d:%d" % (
-                        glyph, otherLoc[0], otherLoc[1], otherLoc[2]),
-                    definition.location)
+# Fixing this code fails tests!
```

Comment:
This is a bigger issue than this PR, I feel. For example, should testdata/markClass.fea fail? I don't want to cause a regression with this PR. But highlighting that that code branch never worked, is OK by me. The point is that I haven't actually changed anything, since that code branch never worked. I think it's a different bug to decide whether we want it to work.